pkgs/applications/video/mpv/scripts/modernx-zydezu.nix

```
@@ -7,14 +7,14 @@
 }:
 buildLua (finalAttrs: {
   pname = "modernx-zydezu";
-  version = "0.2.9";
+  version = "0.3.2";
 
   scriptPath = "modernx.lua";
   src = fetchFromGitHub {
     owner = "zydezu";
     repo = "ModernX";
     rev = finalAttrs.version;
-    hash = "sha256-Tq/v8EbojvOSSwbu8kdbLVxZWZNi1UH6zAPTcsUcgog";
+    hash = "sha256-iFIkYdTWxKO/UnN187p12/VeKHcGdPw5IN5sr7n1vHo=";
```

```suggestion
    hash = "sha256-RMvIh2cJrwAuWB+uOZ/JkEHyn7PjimlUZPeUb1+C118=";
```
hash mismatch on aarch64-linux. this tag might have been force pushed

---

Didn't test it for `aarch64-linux`, but if you tested it with the new hash it's ok to me.

@getchoo  Should I commit the suggestion or squash it to the initial commit?